frontend/src/components/elements/BokehChart/BokehChart.tsx

```diff
@@ -18,22 +18,22 @@ import React, { ReactElement, useEffect, useCallback } from "react"
 import withFullScreenWrapper from "src/hocs/withFullScreenWrapper"
 import { BokehChart as BokehChartProto } from "src/autogen/proto"
 
+// bokeh doesn't play well with babel / cra (https://github.com/bokeh/bokeh/issues/10658)
+// have consumers provide their own bokeh minified implementation
+// bokeh.esm is renamed because has "import main from “./bokeh.esm.js"
+import Bokeh from "./vendor/bokeh.esm"
```

Comment:
Yes, thank you for adding this comment! Can you reword it for clarity? Let's be as explicit as possible, so that future readers can understand this decision.
- The first line: "We import Bokeh from a vendored source file, because it doesn't play well with Babel (...)"
- The second line: I don't understand what this means. Is it a complete sentence or part of the previous sentence? Can it be reworded so that someone who has no context can understand?
- Same with the third line - I don't understand what it means. "bokeh.esm is renamed" - renamed from what? (Presumably this means that _we've_ renamed from the file, but let's make that explicit.) Why does "import main from ..." require renaming? What's the problem being solved?